pool/connection_pool.go

```diff
@@ -81,6 +81,8 @@ type Opts struct {
 	CheckTimeout time.Duration
 	// ConnectionHandler provides an ability to handle connection updates.
 	ConnectionHandler ConnectionHandler
+	// BalancingMethod is how connections for the request will be selected
```

Comment:
```suggestion
	// BalancingMethod determines how connections for the request will be selected.
```
Please make sure that all new comments end with a dot (this is part of a Tarantool code style).